packages/core/content-type-builder/server/services/schema-builder/content-type-builder.js

```diff
@@ -207,6 +207,10 @@ module.exports = function createComponentBuilder() {
         const attribute = newAttributes[key];
 
         if (isRelation(attribute)) {
+          if (['manyToMany', 'oneToOne'].includes(attribute.relation)) {
+            attribute.dominant = true;
```

Comment:
Do we have some documentation describing this dominant boolean inside attributes? I think its missing on our doc https://docs.strapi.io/developer-docs/latest/development/backend-customization/models.html#model-attributes

Reply:
No the `dominant` attribute is only an internal and temporary variable (used during the creation/edit process).
In the model schema the information already exists with `inversedBy` and `mappedBy`. One side of the relation has `inversedBy` and the other side has `mappedBy`. We can consider that the side with the `inversedBy` is the dominant.

Reply:
Ok ! Thanks for the explanation @petersg83 🤗

Reply:
with pleasure 😊